core/src/test/scala/examples/DummyConsumer.scala

```diff
@@ -24,6 +26,11 @@ object Streams {
   }
 }
 
+// tbd. What is the purpose of this?
```

Comment:
This example reads from topic `dummy`, processes messages and commit offset into kafka after processing. This provides at least once delivery guarantee.

Also this example show how to perform grace shutdown.